onedal/decomposition/pca.cpp

```diff
@@ -24,16 +23,18 @@ namespace decomposition {
 struct params2desc {
     template <typename Float, typename Method, typename Task>
     auto operator()(const pybind11::dict& params) {
-        using namespace dal::pca;
-
         const auto n_components = params["n_components"].cast<std::int64_t>();
```

Comment:
We should be able to support either int or float here

Reply:
it supports [None, "mle", int, float], but the n_components passed to oneDAL is int after processing.